miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java

```diff
@@ -242,4 +249,43 @@ public PoolDto updatePool(@PathVariable Long poolId, @RequestBody PoolDto pool)
     return "[" + LimsUtils.join(names, ",") + "]";
   }
 
+  @RequestMapping(value = "/picker/search")
+  @ResponseBody
+  public PoolPickerResponse getPickersBySearch(@RequestParam("platform") String platform, @RequestParam("query") String query)
+      throws IOException {
+    PlatformType platformType = PlatformType.get(platform);
+    return getPoolPickerWithFilters(100,
+        PaginationFilter.platformType(platformType),
+        PaginationFilter.query(query));
```

Comment:
It might make sense to have this support the usual search using `PaginationFilter.parse`, but I'm indifferent.